rosdep/python.yaml

```diff
@@ -1386,6 +1386,13 @@ python-freezegun-pip:
   ubuntu:
     pip:
       packages: [freezegun]
+python-ftdi1:
+  debian: [python-ftdi1]
+  gentoo: [dev-embedded/libftdi]
+  ubuntu:
```

Comment:
Please collapse this to a single line rule. It is available on all currently supported distros since xenial there's no need to call out specific versions. 
https://packages.ubuntu.com/xenial/python-ftdi1 (The website only lists supported distros, intermediate ones are certainly there. And likely some of the slightly older ones.) It's not available on Trusty but since the rule doesn't exist, it's known that no-one is using it on trusty, so having an invalid rule won't break anyone.

Reply:
Okay, if the non-existing rule for trusty is not a problem...

Reply:
> It's known that no-one is using it on trusty, so having an invalid rule won't break anyone.

@tfoote This seems contrary to our general position on rosdep validity. For other recent changes we've used the

```yaml
key:
  ubuntu:
    '*': [debname]
    trusty: null
```   

construct as we do for [libdlib-dev](https://github.com/ros/rosdistro/blob/bacb693a5f092c7d844cfda522321075a957d409/rosdep/base.yaml#L1635-L1637). Has our general policy changed with Trusty's imminent retirement or was I misinterpreting things previously?

Reply:
The `'*'` syntax is a little bit more correct. I'd be fine with either. With trusty EOL in a few months, saving a future touch of the file to remove the older rule at some point has some benefits. The wildcard rule will still fail for recently EOL'd distros. Anything not completely specific is prone to overmatch for some elements.